web_client/js/views/visualization.js

```diff
@@ -284,7 +285,21 @@ histomicstk.views.Visualization = girder.View.extend({
             attribution: '',
             tileWidth: 256,
             tileHeight: 256,
-            tileRounding: Math.ceil
+            tileRounding: Math.ceil,
+            tilesAtZoom: function (level) {
+                var scale = Math.pow(2, opts.maxLevel - level);
+                return {
+                    x: Math.ceil(opts.sizeX / opts.tileWidth / scale),
+                    y: Math.ceil(opts.sizeY / opts.tileHeight / scale)
+                };
+            },
+            tilesMaxBounds: function (level) {
```

Comment:
This callback is not getting called in the client test.  Are there any special conditions necessary for geojs to use it?